src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerDriver.CompilationData.cs

```diff
@@ -13,22 +13,24 @@ internal abstract partial class AnalyzerDriver : IDisposable
     {
         internal class CompilationData
         {
-            private readonly Dictionary<SyntaxReference, DeclarationAnalysisData> _declarationAnalysisDataMap;
+            private readonly record struct SyntaxReferenceAndSymbol(SyntaxReference Declaration, ISymbol Symbol);
```

Comment:
>SyntaxReferenceAndSymbol [](http://example.com/codeflow?start=43&length=24)

Is this new type used?

Reply:
Ah thanks, I switched to triple later. Will delete.